vlib/v/checker/tests/generic_interface_method_decl_err.vv

```diff
@@ -1,4 +1,4 @@
-interface Expr {
+interface Expr<R> {
```

Comment:
I am curious, why use `interface Expr<R> {`, instead of the newer `interface Expr[R] {` ?

Reply:
Ok, I'll cleanup them.